src/lib.rs

```diff
@@ -91,6 +91,8 @@ pub struct Config {
     shared_flag: Option<bool>,
     static_flag: Option<bool>,
     check_file_created: bool,
+    warnings_into_errors: bool,
+    warnings: bool,
```

Comment:
I'd like to be able to turn off -Wextra only, many projects compile with -Wall just fine, but they don't compile silently with -Wextra, and this is on purpose.